VALUES.md

```diff
@@ -0,0 +1,35 @@
+
+Ayo.js wants to go further than Node Core did in spreading responsibility across the project, so everyone can focus and specialize better than before. They should be able to organize and manage their own subprojects in a way that is effective and efficient for their individual unique needs.
+
+## Humans are more important than red tape.
+
+If governance policy is not serving the values set forth here, that governance policy should change with the required agility. Following the governance policy to the letter should not take precedence over staying true to our stated values, not to mention the well-being of our humans.
+
+## Humans are more important than APIs and Benchmarks.
+
+Technical discussions are difficult, especially in large communities with members with all sorts of perspectives. You're bound to find yourself in situations with no clear consensus, with new concerns being brought up on a regular basis, and things that might seem insignificant in other contexts suddenly explode.
+
+The Ayo.js project's perspective is that when this happens, it often takes a toll on the well-being of the members participating, or it prevents potential members from contributing again, if at all. It also recognizes that the difference between individual choices is less important than the ability to come to a perfect technical decision.
+
+Projects change, evolve, and find themselves dealing with new concerns all the time. In all technical matters, Ayo.js will take into account that in the end, it's the people making those decisions that matter most, not the decisions themselves. It's ok for code to be imperfect.
+
+## Humans are more important than corporations.
+
+All sorts of corporations and other organizations around the world use and benefit from the software underlying Ayo.js. Many contributors, in fact, might have their efforts in the project directly funded by them. But, this can often turn into such direct reliance on corporations that they can essentially seize control of a project -- or struggle for power over it while individual contributors are left wondering if they can ever influence the project themselves.
+
+Ayo.js will explore new structures and framings for corporate relationships in such a way that its individual humans, not the corporations that employ them, are ultimately still the ones determining the direction of the project. Ayo.js should be a space where independent contributors, freelancers, hobbyists, and students all feel like they hold the same footing as corporate employees at all levels of the project.
+
+## Awoo <!-- not keeping this here unless y'all really want it but it's making me happy to see here for now -Hal -->
```

Comment:
i vote that we ACTUALLY remove this, especially since awoo is already mentioned at the top, and is therefore already part of our brand